client/src/components/Podiums/Podiums.js

```diff
@@ -75,6 +83,29 @@ const Podiums = ({ match }) => {
               />
             </Grid>
           ))}
+          {competition.events.length !== competition.podiums.length && (
+            <Grid item key={`events_not_completed`}>
+              <Typography>{`Podiums for the following events are yet to be determined:`}</Typography>
+              <Grid container direction="row" spacing={3}>
+                {competition.events
+                  .filter(
+                    event =>
+                      competition.podiums.filter(
+                        podium => podium.event.id === event.id
+                      ).length === 0
```

Comment:
As we test for the presence only, `.filter(...).length === 0` could be replaced with `.some(...)` :cat: